### PR TITLE
Add Nepali news classification dataset

### PR DESCRIPTION
**Pull request by bp-high**
## Checklist for adding MMTEB dataset
Reason for dataset addition:
- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Comment by isaac-chung** [comment]
Marking this as draft. Feel free to convert it back when you have gone through the PR checklist.

File: mteb/tasks/Classification/nep/NepaliNewsClassification.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    abstract = "We present iNLTK, an open-source NLP library consisting of pre-trained language models and out-of-the-box support for Data Augmentation, Textual Similarity, Sentence Embeddings, Word Embeddings, Tokenization and Text Generation in 13 Indic Languages. By using pre-trained models from iNLTK for text classification on publicly available datasets, we significantly outperform previously reported results. On these datasets, we also show that by using pre-trained models and data augmentation from iNLTK, we can achieve more than 95{\%} of the previous best performance by using less than 10{\%} of the training data. iNLTK is already being widely used by the community and has 40,000+ downloads, 600+ stars and 100+ forks on GitHub.",
+    }
+    """,
+        n_samples={"train": 5975, "test": 1495},
+        avg_character_length={"train": 196.61, "test": 196.017},
+    )
+
+    def dataset_transform(self):
+        self.dataset = self.dataset.rename_column("paras", "text")
+        self.dataset["train"] = self.dataset["train"].select(range(2048))
```
Comment (isaac-chung) [inline mteb/tasks/Classification/nep/NepaliNewsClassification.py:57]:
```suggestion
self.dataset = self.stratified_subsampling(
    self.dataset, seed=self.seed, splits=["train"]
)
```

**Review by isaac-chung — APPROVED** [review]
Thanks for adding this! Just a small comment.
And feel free to add 2+4=6 points for yourself and 2 pts for me for the review.

File: mteb/tasks/Classification/nep/NepaliNewsClassification.py (hunk 1 above)
Comment (bp-high) [reply mteb/tasks/Classification/nep/NepaliNewsClassification.py]: Done

**Pull request merged**